backend/crypto.h

```diff
@@ -1,6 +1,7 @@
 #ifndef CRYPTO_H_
 #define CRYPTO_H_
 
+extern const int KEY_LENGTH;
```

Comment:
No need to declare constant here if it is used only in crypto.cpp (definition there is enough)